### PR TITLE
Adding tmpl-specs

### PR DESCRIPTION
**Pull request by dustyo-O**

### CODE REVIEW

File: .enb/make.js [hunk 1]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
 module.exports = function(config) {
     config.setLanguages(['en', 'ru']);
 
+    config.includeConfig('enb-bem-tmpl-specs'); // Подключаем `enb-bem-tmpl-specs` модуль.
+
+    var examples = config.module('enb-bem-tmpl-specs') // Создаём конфигуратор сетов
+        .createConfigurator('tmpl-specs', {            // в рамках таска `specs`.
+            coverage: {                                // Определяем общие опции для всех уровней-сетов.
+                engines: ['bemhtml'],
+                reportDirectory: 'coverage',
```
Comment (tadatuta) [inline .enb/make.js]: default value

File: .enb/make.js [hunk 2]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
...
+                exclude: ['**/node_modules/**', '**/libs/**'],
+                reporters: ['lcov']
```
Comment (tadatuta) [inline .enb/make.js]: default value

File: .enb/make.js [hunk 3]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
...
     config.setLanguages(['en', 'ru']);
 
+    config.includeConfig('enb-bem-tmpl-specs'); // Подключаем `enb-bem-tmpl-specs` модуль.
+
+    var examples = config.module('enb-bem-tmpl-specs') // Создаём конфигуратор сетов
+        .createConfigurator('tmpl-specs', {            // в рамках таска `specs`.
+            coverage: {                                // Определяем общие опции для всех уровней-сетов.
+                engines: ['bemhtml'],
+                reportDirectory: 'coverage',
+                exclude: ['**/node_modules/**', '**/libs/**'],
```
Comment (tadatuta) [inline .enb/make.js]: default value

File: .enb/make.js [hunk 4]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
 module.exports = function(config) {
     config.setLanguages(['en', 'ru']);
 
+    config.includeConfig('enb-bem-tmpl-specs'); // Подключаем `enb-bem-tmpl-specs` модуль.
+
+    var examples = config.module('enb-bem-tmpl-specs') // Создаём конфигуратор сетов
+        .createConfigurator('tmpl-specs', {            // в рамках таска `specs`.
+            coverage: {                                // Определяем общие опции для всех уровней-сетов.
+                engines: ['bemhtml'],
```
Comment (tadatuta) [inline .enb/make.js]: check if it's right

File: .enb/make.js [hunk 5]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
...
+        destPath: 'common.tmpl-specs',
+        levels: ['common.blocks'],
+        langs: true,
+        sourceLevels: levels,
+        engines: {
+            'bemhtml-dev': {
+                tech: 'enb-bemxjst/techs/bemhtml',
+                options: {
+                    exportName: 'BEMHTML',
+                    devMode: true
```
Comment (tadatuta) [inline .enb/make.js]: remove

add `engineOptions` (?) with the same values as in `techs.bemhtml` bellow

File: .enb/make.js [hunk 6]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
...
+                }
+            },
+            'bemhtml-prod': {
```
Comment (tadatuta) [inline .enb/make.js]: remove

File: .enb/make.js [hunk 7]
```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
...
+                reportDirectory: 'coverage',
+                exclude: ['**/node_modules/**', '**/libs/**'],
+                reporters: ['lcov']
+            }
+        });
+
+    examples.configure({
+        destPath: 'common.tmpl-specs',
+        levels: ['common.blocks'],
+        langs: true,
```
Comment (tadatuta) [inline .enb/make.js]: remove

File: common.blocks/header/header.tmpl-specs/10-header.html [hunk 8]
```diff
@@ -0,0 +1 @@
+<div class="header"><span class="logo"><svg class="logo__logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 101.7 84"><path d="M0 13h23.8v7H0zM0 0h23.8v7H0zm42.7 25.9H0v7h41.1c4.5 0 15 .1 18.9 6.4v-3.9c-1.9-6.2-7.5-9.5-17.3-9.5zm.1 13H0v7h41.1c10 0 16.3.4 18.9 1.8v-3.1c-2-4.9-10.4-5.7-17.2-5.7zm-.1 33H0v-7h41.1c4.5 0 15-.1 18.9-6.4v3.9c-1.9 6.2-7.5 9.5-17.3 9.5zm.1-13H0v-7h41.1c10 0 16.3-.4 18.9-1.8v3.1c-2 4.9-10.4 5.7-17.2 5.7zM70.4 84h31.4v-4.3H71.3c-2.7 0-9.1-.1-11.4-3.8v2.4C61 82 64.4 84 70.4 84zm-.1-7.9h31.4v-4.3H71.3c-6.1 0-9.9-.2-11.4-1.1v1.9c1.2 3 6.3 3.5 10.4 3.5z"/></svg></span><a class="button button_theme_islands button_size_m button_view_action button_type_link button__control header__button i-bem" data-bem='{"button":{}}' role="link" href="/auth/github"><span class="button__text">Войти с помощью github</span></a></div>
```
Comment (tadatuta) [inline common.blocks/header/header.tmpl-specs/10-header.html]: beautify

File: common.blocks/header/header.tmpl-specs/20-header.bemjson.js [hunk 9]
```diff
@@ -0,0 +1,54 @@
+module.exports = {
+    block: 'header',
+    content: [
+        {
+            block: 'logo',
+            url: false
+        },
+        [
+            {
+            block: 'avatar',
```
Comment (tadatuta) [inline common.blocks/header/header.tmpl-specs/20-header.bemjson.js]: format

**Review by tadatuta — COMMENTED** [review]
add tmpl specs to npm scripts

**Comment by dustyo-O** [comment]
fixed

**Review by tadatuta — APPROVED** [review]

**Pull request merged**